src/actor/module/wasmbus.rs

```diff
@@ -250,18 +249,6 @@ async fn host_call(
     let pld = read_bytes(&mut store, &memory, pld_ptr, pld_len)
         .context("failed to read `__host_call` payload")?;
 
-    ensure!(
-        store
-            .data()
-            .claims
-            .metadata
-            .as_ref()
-            .map(|jwt::Actor { caps, .. }| caps.as_ref().map(|caps| caps.contains(&ns)))
-            .unwrap_or_default()
-            .unwrap_or_default(),
-        "`{ns}` capability request unauthorized"
-    );
-
```

Comment:
Please create followup task so we can investigate ways to do this in this library in the future 👍🏻

Reply:
https://github.com/wasmCloud/wasmCloud/issues/290